content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -952,7 +954,7 @@ applicable:
 * iSCSI
 * Local volume
 * OpenStack Cinder
-* RBD (Ceph Block Device)
+* RBD (Ceph Block Device) ( **deprecated** in v1.28)
```

Comment:
```suggestion
* RBD (deprecated)
```

Reply:
Should we also update the following?
- AWSElasticBlockStore (in-tree plugin is already removed)
- AzureDisk (in-tree plugin is already removed)
- GCEPersistentDisk (in-tree plugin is removed in 1.28)
- OpenStack Cinder (in-tree plugin is already removed)
-  VsphereVolume (deprecated, in-tree plugin will be removed in 1.30)

I don't see AzureFile here, but AzureFile in-tree plugin is also deprecated and will be removed in 1.30 as well.

Reply:
I'm fine for these to be updated in a different PR.

Reply:
@xing-yang would you be willing to file an issue? Fixes against _main_ are welcome and this is almost a good-first-issue.

Reply:
Issue opened: https://github.com/kubernetes/website/issues/42256